<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatServerForm</class>
 <widget class="QWidget" name="ChatServerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chatting Server</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>State</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>IP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ipLineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>Before Connection</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="stateLineedit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Chatting_On</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeWidget" name="chattingTreeWidget">
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string>Status</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="clientListLabel">
         <property name="text">
          <string>Client List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="clientTreeWidget">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="messageVerticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Chatting Log</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="messageTreeWidget">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>IP</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Port</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Message</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="messageLayout">
         <item>
          <widget class="QPushButton" name="clearPushButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearPushButton</sender>
   <signal>clicked()</signal>
   <receiver>messageTreeWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
